CactbotOverlay/CactbotOverlay.cs

```diff
@@ -247,6 +253,79 @@ public CactbotOverlay(CactbotOverlayConfig config)
       fast_update_timer_semaphore_.Release();
     }
 
+    public void EnableWatchers(bool check) {
+      this.watcher_state = check;
+    foreach (var watcher in this.watchers)
+        watcher.EnableRaisingEvents = check;
+    }
+ 
+    private void SetupFileWatcher(string url) {
+      if (url == "")
+        return;
+
+      foreach(var watcher in this.watchers)
+        watcher.Dispose();
+
+      List<string> path_list = new List<string>();
+      var overlay_full_path = url;
+      string base_path = null;
+
+      if (!url.Contains("://quisquous.github.io/cactbot/")) {
```

Comment:
Maybe this should check if it starts with `file://` instead? Folks could use a different location for remote urls.

Reply:
I saw you were using this method somewhere else to determine whether its a local or remote url.
But doesn't the overlay accept local urls lacking the `file:///`-protocol?

Reply:
Hmm, that's fair.  What about checking for http or https instead?